frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -346,7 +357,7 @@ export class CatalogTileViewPage extends React.Component<
                 <div className="co-catalog-page__overlay-actions">
                   <Link
                     className="pf-c-button pf-m-primary co-catalog-page__overlay-action"
-                    to={detailsItem.href}
+                    to={this.appendTopologyRedirect(detailsItem.href)}
```

Comment:
It's just hard coding the URL redirect to topology which means that user can be in any perspective and still he will be redirected to topology. Is this the behavior that we want here? cc: @christianvogt 

Below screencast shows that even if the user is in admin perspective, he is redirected to topology.
![Peek 2020-02-29 01-56](https://user-images.githubusercontent.com/6041994/75584982-0f364d80-5a97-11ea-9816-947e960cf1cb.gif)

Reply:
This can be avoided now that the implementation is changed to use active Perspective.